### DISCUSSION TITLE
Opensearch, Elasticsearch and Varnish configuration while setup

### DISCUSSION THREAD

**Discussion by mucan54** [markshust/docker-magento#932] [Ideas]
Hi Mark,
I'm big fan of your setup and I want to use it for my every projects. But I need varnish and most of the time we need elasticsearch. And  we never need to switch them after setup. So maybe we can add some parameters to online setup command to pick our desired setup. Something like : 

`curl -s https://raw.githubusercontent.com/markshust/docker-magento/master/lib/onelinesetup | bash -s -- magento.test 2.4.6 community --search-engine=elasticsearch --varnish=1 --blackfire=1 --selenium=0`

with those config parameter we can change the docker templates and pick the desired setup.

**Comment by YevhenZvieriev** [comment]
Hi, @mucan54 

If @markshust agrees, I can do this
